readme.md

```diff
@@ -37,6 +38,15 @@ is.number(6);
 //=> true
 ```
 
+Assertions perform the same type checks, but throw errors if the type does not match.
+
+```js
+const {assert} = require('@sindresorhus/is');
+
+assert.string(foo);
+//=> foo is known to be a string
```

Comment:
`//=>` is usually used to show output, so I think the text should just be a normal comment above.